packages/adapter/lib/adapter/adapter.js

```diff
@@ -191,6 +189,7 @@ function Adapter(options) {
     this.logList = [];
     this.aliases = {};
     this.aliasPatterns = [];
+    this.enums = {};
```

Comment:
I'll advocate for using [Maps](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) over plain objects for lookup, but you decide.

Reply:
We have objects everywhere right now ... we can start mixing ... what do the overs think?

Reply:
lets fdo that later. I create an issue